docs/_build/html/_sources/tutorial.rst.txt

```diff
@@ -0,0 +1,198 @@
+####################
+ CytoSnake Tutorial
+####################
+
+*******
+ About
+*******
+
+This tutorial assumes that you have follows the installation steps and
+you are ready to start taking off with CytoSnake!
+
+Cytosnake is a command line interface (CLI) tool that contains a
+multitude of workflows for analyzing morphological features obtained
+from microscopy images of cells.
+
+**********
+ Concepts
+**********
+
+.. image:: images/Cytopipe-cli_0.drawio.png
+
+Modes
+=====
+
+Modes provide options on how the user can change the functionality of
+CytoSnake. For example, if you would like to initialize your files for a
+specific workflow, you can simply type:
+
+using the `init` mode
+
+.. code::
+
+   cytosnake init <DATAFILES> <METADATA> <DATATYPE>
+
+-  `DATAFILE` will refer to the raw data that you are going to analyze,
+-  `METADATA` refers to the associated metadata data directory that was
+   generated along with the dataset
+-  `DATATYPE` flag tells cytosnake weather these morphology feature
+   datasets were obtained from CellProfiler or DeepProfiler
+
+The init mode setups the provided input files into its appropriate file
+structure that accommodates all the workflows available in Cat Snake
```

Comment:
```suggestion
structure that accommodates all the workflows available in CytoSnake
```

Funny typo 😹 

Also, are you camelcasing CytoSnake, or using all lowercase? I've seen both. Please be consistent.

Reply:
Yikes hahaha. 

Camel casing